### PR TITLE
Seccon 7430

### LINKED ISSUE #10: Missing required modules in stix shifter/basic_df_analysis notebook
Missing the following dependency modules when running 

from pyclient.stix_shifter_dataframe import StixShifterDataFrame

!pip install pandas
!pip install anytree
from pyclient.stix_shifter_dataframe import StixShifterDataFrame

Not sure if dependancies are an issue or what.

### PR DESCRIPTION
**Pull request by charliewutw1**
This PR contains the following changes:
- add a new stix-shifter notebook called cbr-hunt-certutil-using-cb-stix-bundle 
- handle empty dataframe issue for stix_shifter_dataframe
- handle self assigned node in flatten_sco
- fix #10

### CODE REVIEW

**Review by svajipay — APPROVED** [review]

File: pyclient/stix_shifter_dataframe.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def stix_shiter_execute(self, config_name: str, stix_query: str):
                         logging.debug(status)
                     else:
                         raise RuntimeError("Fetching status failed")
-                result = transmission.results(search_id, 0, 9)
+                # TODO: allow user to pass No. of records
```
Comment (pcoccoli) [inline pyclient/stix_shifter_dataframe.py:59]: Can you create an issue for this TODO?

File: pyclient/stix_shifter_dataframe.py [hunk 2]
```diff
@@ -81,8 +82,9 @@ def obj2df(obj):
         dfs = []
         for ldf in list(map(lambda obj: obj2df(obj), stix['objects'])):
             dfs.extend(ldf)
-        df = pd.concat(dfs)
-        return df
+
+        #df = pd.concat(dfs)
```
Comment (pcoccoli) [inline pyclient/stix_shifter_dataframe.py]: Should probably remove commented out code.

File: pyclient/stix_shifter_dataframe.py [hunk 3]
```diff
@@ -111,11 +116,15 @@ def flatten_sco(self, obj, viewname='observed-data'):
                         nodes[val].prefix = attr
                 elif attr.endswith('_refs'):
                     for i, ref in enumerate(val):
+                        if ref == k:
+                            continue
```
Comment (pcoccoli) [inline pyclient/stix_shifter_dataframe.py:120]: A comment here about preventing circular references might be nice.

File: pyclient/stix_shifter_dataframe.py [hunk 4]
```diff
@@ -111,11 +116,15 @@ def flatten_sco(self, obj, viewname='observed-data'):
...
                         nodes[ref].parent = nodes[k]
                         nodes[ref].prefix = attr + f'[{i}]'
                 if attr == 'type':
-                    assert val
-                    assert val != ''
+                    # Ignore this obj
+                    continue
+                    # assert val
+                    # assert val != ''
```
Comment (pcoccoli) [inline pyclient/stix_shifter_dataframe.py]: Should probably remove the commented out code

File: pyclient/stix_shifter_dataframe.py (hunk 1 above)
Comment (charliewutw1) [reply pyclient/stix_shifter_dataframe.py:59]: https://github.com/IBM/ibm-security-notebooks/issues/12

**Review by pcoccoli — APPROVED** [review]

**Pull request merged**